homeassistant/components/reisingerbridge/translations/de.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please don't include additional translations. Limit it to `en.json`. Additional translations can be provided via our Lokalise translation platform after the PR has been merged.

Remove this file for now.